.github/workflows/swift-toolchain.yml

```diff
@@ -1137,15 +1143,19 @@ jobs:
           variant: sccache
 
       - uses: nttld/setup-ndk@v1
+        if: inputs.build_os != 'Windows' || inputs.build_arch != 'arm64'
```

Comment:
Could you factorize this (and others) to this form?
```
!(inputs.build_os == 'Windows' && inputs.build_arch == 'arm64)'
```

It may be just me but I find it easier to infer that we want to run this on everything except for Windows-Arm64 with the negation up front.

Reply:
I don't think that it is just you - that is clearer as to the intent for me as well.

Reply:
Done